Gems/Atom/Feature/Common/Code/Source/ImGui/ImGuiPass.cpp

```diff
@@ -102,7 +104,6 @@ namespace AZ
             Init();
             ImGui::NewFrame();
```

Comment:
Is this NewFrame() still needed?

Reply:
Unfortunately yes; I originally tried removing it but it results in a debug assert firing at startup on the first frame. We probably need an overhaul of ImGui so that it's concept of a frame is better in sync with the rest of the engine, right now we basically have this situation:

```
---- startup ---- 
NewFrame
---- frame start ---- 
<n-possible Render calls> (where n>=0)
EndFrame
NewFrame
```
and this fix is to account for the case where n=0, otherwise we end up calling NewFrame twice in a row with no call to EndFrame in between, which results in the assert (if n>=1, the explicit call I added to EndFrame is benign and does nothing).